src/index.json

```diff
@@ -17133,6 +17133,56 @@
                     "version": "0.26.0"
                 },
                 "sha256Digest": "565fc207f1740c26957f382fe2eefabec254011fb2d1b50c0e540f894f47dcbe"
+            },
+            {
+                "downloadUrl": "https://github.com/Azure/azure-devops-cli-extension/releases/download/20240206.1/azure_devops-1.0.0-py2.py3-none-any.whl",
+                "filename": "azure_devops-1.0.0-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.minCliCoreVersion": "2.30.0",
+                    "classifiers": [
+                        "Development Status :: 4 - Beta",
+                        "Intended Audience :: Developers",
+                        "Intended Audience :: System Administrators",
+                        "Programming Language :: Python",
+                        "Programming Language :: Python :: 3",
+                        "Programming Language :: Python :: 3.4",
+                        "Programming Language :: Python :: 3.5",
+                        "Programming Language :: Python :: 3.6",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "VSTS_Social@microsoft.com",
+                                    "name": "Microsoft",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Microsoft/azure-devops-cli-extension"
+                            }
+                        }
+                    },
+                    "extras": [],
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "azure-devops",
+                    "run_requires": [
+                        {
+                            "requires": [
```

Comment:
If you download https://github.com/Azure/azure-devops-cli-extension/releases/download/20240206.1/azure_devops-1.0.0-py2.py3-none-any.whl and open "\azure_devops-1.0.0-py2.py3-none-any\azure_devops-1.0.0.dist-info\metadata.json", you can find metadata in whl file is different from the metadata in this PR.


![image](https://github.com/Azure/azure-cli-extensions/assets/16348853/84ffcaed-7287-44ec-a23b-e18e52905a0d)

![image](https://github.com/Azure/azure-cli-extensions/assets/16348853/0a34b986-3e67-4d86-9244-0fb38a0f9142)

The CI error message also shows the root cause

![image](https://github.com/Azure/azure-cli-extensions/assets/16348853/aebb7816-c8d9-441c-bbaf-7fe892323528)